Gruntfile.js

```diff
@@ -38,13 +39,18 @@ module.exports = function(grunt) {
 					{ collapseGroups: true },
 					{ removeEmptyAttrs: true },
 					{ removeUselessStrokeAndFill: true },
-					{ removeViewbox: false }
+					{ removeViewbox: false },
+					{
+						removeAttrs: {
+							attrs: ['fill']
+						}
+					}
 				]
 			},
 			dist: {
 				files: [{
 					expand: true,
-					cwd: 'src/svg',
+					cwd: 'dist/svg/raw',
```

Comment:
This doesn't match the `dist/` specified in the const above. Let's keep this PR as minimal as we can, and keep dist reorganization as a separate task.

svg => `dist/svg`
optimized => `dist/optimized`
sprite => `dist/sprite`
pdf => `dist/pdf`